docs/tutorials/mobile/part-1.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Part 1: Create the smart contract'
+authors: 'Benjamin Fuentes'
+last_update:
+  date: 12 December 2023
+---
+
+On this first section, you will :
+
+- create the game smart contract importing an existing Ligo library
+- deploy your smart contract to the Ghostnet
+- get the Shifumi Git repository folders to copy the game UI and CSS for the second party
+
+## Smart contract
```

Comment:
What does the game contract do? What logic does it contain and what data does it store? Give me an overview of how this game application works, like info about what the frontend and backends do.

Reply:
The goal of the tutorial is not to detail the game logic but how to create a mobile application. Also there are complex parts on this game that requires another tutorial just for it